pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "up42-py"
-version = "0.33.1"
+version = "0.33.2"
```

Comment:
@iliascholl-up42 
I completely forgot.
We don't need to bump version since we are not releasing a new version. We can release it with a bunch of other PRs related to v2 endpoint migration.

Or, do you want to release a new SDK version for this change?

Reply:
no, we don't need to release this one, I think. In the worst case, the customer will try to get assets for source `ANALYTICS` and encounter a bad request. The chance is rather low, I think.
I'll revert the change of the version then